### PR TITLE
[WIP] Add support for raw images and images refs in md files

### PR DESCRIPTION
**Pull request by RadioPotin**
Add support for raw image files in Sources tab, working on fixing their reference in markdown files !

### CODE REVIEW

File: src/htmlize/main.ml [hunk 1]
```diff
@@ -52,6 +52,11 @@ let htmlize filename content =
     Printf.bprintf b {|%s|} content;
     Buffer.contents b
 
+  | "jpeg" | "jpg" | "png" | "gif" | "tiff" | "raw" ->
```
Comment (zapashcanon) [inline src/htmlize/main.ml:55]: JPG JPEG svg ico

Comment (zapashcanon) [reply src/htmlize/main.ml:55]: Here's [a list](https://developer.mozilla.org/fr/docs/Web/Media/Formats/Image_types) :)

Comment (RadioPotin) [reply src/htmlize/main.ml:55]: That has already been done at line 48 !

File: src/htmlize/patchtml.ml [hunk 2]
```diff
@@ -138,3 +137,6 @@ let rec handle_block = function
 
 let handle_file doc  =
   List.map handle_block doc
+
+let render_img img extension =
+  Format.sprintf "<img type=\"image/%s\" src=\"raw/%s\"/>" extension img
```
Comment (zapashcanon) [inline src/htmlize/patchtml.ml:142]: remove `type`

File: src/htmlize/patchtml.ml [hunk 3]
```diff
@@ -41,7 +41,7 @@ let is_anchor link =
  * Then prepend the corresponding number of .. to the basename
  * Then append index.html to fix link
  * *)
-let handle_link link =
+let handle_link ~is_img:is_img link =
```
Comment (zapashcanon) [inline src/htmlize/patchtml.ml:44]: `is_raw`

File: test/patchtmltests.ml [hunk 4]
```diff
@@ -30,9 +30,35 @@ let () =
...
+    let result = Htmlize.Patchtml.handle_link ~is_img:false sent in
     let ok = result = expected in
     if not ok then Format.printf "expected: %s but got %s@." expected.destination sent.destination;
 
     assert ok
-  ) test_cases
+  ) test_cases;
+
+  let img_test_cases = [|
+    (".", "raw/.");
```
Comment (zapashcanon) [inline test/patchtmltests.ml:41]: more tests :-)

File: src/htmlize/patchtml.ml [hunk 5]
```diff
@@ -50,6 +50,11 @@ let handle_link link =
     link
 
   else
+    let target =
+      match is_raw with
```
Comment (zapashcanon) [inline src/htmlize/patchtml.ml:54]: `if is_raw ...`

**Pull request merged**